lib/common/multithread.c

```diff
@@ -87,11 +87,16 @@ static void init_async(h2o_multithread_queue_t *queue, h2o_loop_t *loop)
 {
     int fds[2];
 
+#ifndef _WIN32
     if (cloexec_pipe(fds) != 0) {
         perror("pipe");
         abort();
     }
     fcntl(fds[1], F_SETFL, O_NONBLOCK);
+#else
+    u_long nonblock = 1;
```

Comment:
> I agree that porting the libuv-side would be faster, but if your intention is to make the H2O standalone server runnable on Windows, you would need to port evloop (and it seems like that you have already done that to some extent!).

Yes I started working on that because I couldn't get the examples that use libuv working :(
I would like to get libh2o working first if possible because I want to create a few mockups with it.

I still intend to get the full package working on windows I would like to focus on the lib aspect first.